core/src/main/scala/anorm/Column.scala

```diff
@@ -477,40 +487,44 @@ object Column extends JodaColumn with JavaTimeColumn {
     def transf(a: Array[_], p: List[T]): Either[SqlRequestError, List[T]] =
       a.headOption match {
         case Some(r) => transformer(r, meta) match {
-          case Right(v) => transf(a.tail, p :+ v)
+          case Right(v) => transf(a.tail, v :: p)
```

Comment:
I'm impressed with your attention to detail, fixing up this, and changing `Seq[T]()` to `Seq.empty[T]`!

Reply:
Or did this plugin help? https://github.com/sksamuel/sbt-scapegoat :)

Reply:
That's it